virttest/utils_sys.py

```diff
@@ -74,6 +75,33 @@ def get_host_bridge_id(session=None):
     return host_bridges if host_bridges else []
 
 
+def is_image_mode(check_command="rpm-ostree", session=None):
+    """
+    Check if current OS is in image mode or package mode
+
+    :param check_command: str, the command to check the mode
+    :param session: aexpect session
+    :return: boolean, True for image mode, otherwise False
+    """
+    if session:
+        check_command = "which %s" % check_command
+        status = session.cmd_status(check_command)
+        ret = status == 0
+    else:
+        try:
+            utils_path.find_command(check_command)
```

Comment:
I don't think it is a reliable way to check if it is image mode because rpm-ostree could be installed in image mode as well. The better way is via bootc:
- No bootc cmd --> package mode
- With boot, then check `bootc status`
  - Return **System is not deployed via bootc** --> package mode
  - Else --> image mode